.github/workflows/ci_test-full.yml

```diff
@@ -21,6 +21,9 @@ jobs:
         python-version: ["3.7", "3.9"]  # minimum, maximum
         requires: ["oldest", "latest"]
         release: ["stable"]
+        exclude:
+          # Skip if torch<1.8 and py3.9 on Linux: https://github.com/pytorch/pytorch/issues/50014
+          - {os: ubuntu-18.04, python-version: "3.9", requires: "oldest", release: "stable"}
```

Comment:
btw, I would not run any oldest for py3.9
```yaml
- {python-version: "3.9", requires: "oldest"}
```

Reply:
@Borda Could I ask why you wouldn't? This problem shouldn't apply to other oldest settings.

Reply:
use `release: "stable"` is redundant since we do not have other release values then stable now
and in general, if someone is using almost the latest python I don't think he would assist on the oldest packages...

Reply:
> use `release: "stable"` is redundant since we do not have other release values then stable now

:+1: 

> in general, if someone is using almost the latest python I don't think he would assist on the oldest packages...

I see your point. I've just created an issue #11367 to gain some visibility on this.